configs/material-ui.json

```diff
@@ -1,7 +1,14 @@
 {
   "index_name": "material-ui",
   "start_urls": [
-    "https://material-ui.com/"
+    {
+      "url": "https://material-ui.com/",
+      "version": "v3"
```

Comment:
This is not the correct way to add version within your pages. You can use `tags`. 
Please implement the meta tags as highlighted in the next review instead.

Reply:
Should I use variables instead?
https://community.algolia.com/docsearch/config-file.html#using-regular-expressions

Reply:
Please use the meta tags instead.